2018/1/solution.py

```diff
@@ -0,0 +1,25 @@
+import sys
```

Comment:
(nit) this might just be a personal preference, but it wasn't clear to me what frequency_total and frequency_duplicate did if I just read the function name, but didn't know the problem.  I think it's because I prefer functions to be a verb